src/sdk/clients/decorators/erc7579/modules/ownableValidator/addOwner.ts

```diff
@@ -0,0 +1,86 @@
+import { getAddOwnableValidatorOwnerAction } from "@rhinestone/module-sdk"
```

Comment:
Can you move this whole decorators folder into `modules/validators/ownableValidator?` I think it makes more sense there.